CHANGELOG.md

```diff
@@ -10,10 +10,16 @@ Current
 
 ### Added:
 
+- [Support for Druid's In Filter](https://github.com/yahoo/fili/pull/64)
+
 - [Documentation for topN](https://github.com/yahoo/fili/pull/43)
 
 ### Changed:
 
+- [Cleaned up the type for `DimensionalFilter::withDimension`](https://github.com/yahoo/fili/pull/64)
+    -- Previously, `DimensionalFilter::withDimension` had the signature `public abstract <T> T withDimension(Dimension dimension)`,
+    but `T` is already defined as part of the class type of `DimensionalFilter`. Therefore, `<T>` has been removed.
```

Comment:
Does that actually change anything? If not (ie. if it's the same as changing `List<Fool> f = new ArrayList<Foo>()` into `List<Fool> f = new ArrayList<>()`), then I wouldn't bother noting it in the change log.